README.md

```diff
@@ -28,7 +28,7 @@ The `.deb` that Tauri builds will automatically do this for us for actual toolki
 1. (New Method) Run the following command.
 
     ```
-    $ curl -sSL https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/blob/main/install-update-media/install-ddt.py -o install-ddt.py && python3 install-ddt.py
+    $ curl -sSL https://raw.githubusercontent.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/blob/main/install-update-media/install-ddt.py -o install-ddt.py && python3 install-ddt.py
```

Comment:
Remove the /blob/ so that it just reads .../main/...

![image](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/assets/62839826/ccfebc61-2bcc-4f9d-a308-fb8191fe05e4)
![image](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/assets/62839826/8dbd98f3-9b29-4208-b589-603edf3b25f5)